File: 4_Matricies/max/max.c
void findMatrixMax(int matrix[4][4], const int SIZE, int * max) {
    max[0] = 0, max[1] = 0;
    for (int y = 0; y < SIZE; y++) 
        for (int x = 0; x < SIZE; x++) 
            if (matrix[y][x] > matrix[max[0]][max[1]]) {
                max[0] = y; 
                max[1] = x;
            }
}

File: 4_Matricies/max/max.js
// finds max number of matrix
function findMax(matrix) {
    let max = Array(2);
    (max[0] = 0), (max[1] = 0);
    for (let y = 0; y < matrix.length; y++) {
      for (let x = 0; x < matrix.length; x++) {
        if (matrix[y][x] > matrix[max[0]][max[1]]) {
          max[0] = y;
          max[1] = x;
        }
      }
    }
    return max;
  }